src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -1963,5 +1963,286 @@ public void SarifTransformerTests_ToVersionOne_Regions()
 
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_CodeFlows()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
```

Comment:
This is optional. Let's make our tests a little shorter by omitting it.

Reply:
Done

Reply:
Actually tool is required.

Reply:
Correct, but `tool.semanticVersion` is not.

Reply:
I was not sufficiently precise in my original comment.

Reply:
OK